tpgtools/api/privateca/samples/basic_certificate_template.yaml

```diff
@@ -1,21 +1,27 @@
 # Copyright 2021 Google LLC. All Rights Reserved.
-#
+# 
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
-#
+# 
 #     http://www.apache.org/licenses/LICENSE-2.0
-#
+# 
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-dependencies:
-- samples/default.run.service.json
-description: A basic example for an Eventarc Trigger
-name: basic_trigger
-resource: samples/trigger.eventarc.trigger.json
+description: An example of a basic privateca certificate template
+name: basic_certificate_template
+resource: samples/basic.certificate_template.json
+updates:
+- resource: samples/update.certificate_template.json
+variables:
+- name: template
+  type: resource_name
+- name: project
+  type: project
+- name: region
+  type: region
 versions:
-- beta
-- ga
\ No newline at end of file
+- ga
```

Comment:
Recommend adding a handwritten terraform test against beta with an update... as is there is no test running against beta

Reply:
thus no test for vcr

Reply:
Added the existing test to beta since the beta is the same as ga!